js/utils.ts

```diff
@@ -44,3 +45,28 @@ export function reverseMap<K, V>(map: Map<K, V>): Map<V, K> {
     }
     return reversedMap;
 }
+
+export interface SelectOption {
+    label: string;
+    value: string;
+}
+
+export function renderSelect(
```

Comment:
If it's not too much to ask, can you refactor the basemap selector to use this please?